Documentation/community/community.rst

```diff
@@ -41,6 +41,8 @@ Name                 Purpose
 #kubernetes          Kubernetes-specific questions
 #networkpolicy       Questions on network policies
 #release             Release announcements only
+#service-mesh        Questions on Cilium Service Mesh
```

Comment:
a lot of community members ares using this slack channel for clustermesh as well, seems like we don't have dedicated channel for clustermesh.

Reply:
Can you be more explicit please, is this an request for creating the channel and adding to the list in this PR? Or are you OK with the current update?
@lizrice: Have you considered creating #clustermesh?

Reply:
yeah, it will be great to have a dedicated clustermesh channel :+1:, the changes in this PR look good to me.

Reply:
OK thanks! Now with this PR, I guess you know what to do to request the channel creation :grin: